lib/core/logger.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/foundation.dart';
+import 'package:logger/logger.dart';
+
+final logger = Logger(
+  level: kDebugMode ? Level.trace : Level.off,
+  printer: PrettyPrinter(
+    methodCount: 5,
+    printTime: true,
+  ),
```

Comment:
TODO : Add an output parameter which sends messages to :
- in debug mode : stdout and file
- in production mode : only file

The generated files should be purged after a certain time.

Reply:
je vais créer une issue pour ça parce que flemme de le faire maintenant mais il faudra avant la v1

Reply:
ptdr tu as déjà créé une issue, très bien